hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestReplicationStatus.java

```diff
@@ -139,6 +139,7 @@ private List<ReplicationLoadSource> waitOnMetricsReport(int greaterThan, ServerN
       metrics.getLiveServerMetrics().get(serverName).getReplicationLoadSourceList();
     while (list.size() <= greaterThan) {
       Threads.sleep(1000);
+      list = metrics.getLiveServerMetrics().get(serverName).getReplicationLoadSourceList();
```

Comment:
This is still incorrect...

We need to call hbaseAdmin.getClusterMetrics every time when checking...

And here we'd better use the waitFor method to limit the max wait time?

Reply:
Thanks for the review. @Apache9
Sorry, I made an obvious mistake.